src/plugins/loan-channel-query-map.js

```diff
@@ -10,7 +10,7 @@ export default {
 				{
 					id: 5,
 					url: 'women',
-					queryParams: 'status=fundRaising&gender=female&riskRating=0,5',
+					queryParams: 'status=fundRaising&gender=female',
```

Comment:
As noted below, I've removed all the default riskRating query params leaving only those that alter the range. Altered riskRatings will be redirected to legacy lend while the rest can be served by flss lend/filter